exporter/kafkaexporter/README.md

```diff
@@ -65,6 +65,8 @@ The following settings can be optionally configured:
 - `producer`
   - `max_message_bytes` (default = 1000000) the maximum permitted size of a message in bytes
   - `required_acks` (default = 1) controls when a message is regarded as transmitted.   https://pkg.go.dev/github.com/Shopify/sarama@v1.30.0#RequiredAcks
+  - `compression` (default = 'none') the compression used when producing messages to kafka. https://pkg.go.dev/github.com/Shopify/sarama@v1.30.0#CompressionCodec
```

Comment:
Let's add the list of options here (users are more likely to read README.md than config.go)